workspaces/marketplace/plugins/marketplace-backend/src/services/MarketplaceService.ts

```diff
@@ -21,5 +21,8 @@ import {
 
 export interface MarketplaceService {
   getPlugins(): Promise<MarketplacePluginEntry[]>;
+  getPluginByName(name: string): Promise<MarketplacePluginEntry>;
   getPluginList(): Promise<MarketplacePluginList[]>;
+  getPluginListByName(name: string): Promise<MarketplacePluginList>;
+  getPluginsByPluginsListName(name: string): Promise<MarketplacePluginEntry[]>;
```

Comment:
Passing a PluginListName, so I would drop one s here :man_shrugging: 

```suggestion
  getPluginsByPluginListName(name: string): Promise<MarketplacePluginEntry[]>;
```